### PR TITLE
Codechef Factorial(FCTRL) Solution in C

### PR DESCRIPTION
**Pull request by ritish099**
**Fixes issue:#4612**

**Changes:**

Added C file for Codechef Factorial(FCTRL) solution 
Added README.md explaining the problem

### CODE REVIEW

File: code/online_challenges/src/codechef/FCTRL/Factorial.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    long t;
+    scanf("%ld", & t);
+    while (t--) 
+    {
+        long n;
+        scanf("%ld", & n);
+        long s = 1, sum = 0;
+        while (1) 
+        {
+            s = s * 5;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/FCTRL/Factorial.c]:
```suggestion
            s *= 5;
```

File: code/online_challenges/src/codechef/FCTRL/Factorial.c [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            if (s > n)
+                break;
+            else {
+                long k;
+                k = n / s;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/FCTRL/Factorial.c]:
```suggestion
                long k = n / s;
```

File: code/online_challenges/src/codechef/FCTRL/Factorial.c [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                sum = sum + k;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/FCTRL/Factorial.c]:
```suggestion
                sum += k;
```

File: code/online_challenges/src/codechef/FCTRL/Factorial.c [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            }
+        }
+        printf("%ld\n", sum);
+        sum = 0;
+        s = 1;
```
Comment (arnavb) [inline code/online_challenges/src/codechef/FCTRL/Factorial.c]:
```suggestion
```

File: code/online_challenges/src/codechef/FCTRL/Factorial.c [hunk 5]
```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+main()
```
Comment (arnavb) [inline code/online_challenges/src/codechef/FCTRL/Factorial.c]:
```suggestion
int main() 
```

**Comment by ritish099** [comment]
Sir @arnavb , look at it. I committed all the changes

**Pull request merged**